modules/api/pkg/handler/v2/routes_v2.go

```diff
@@ -7830,6 +7853,51 @@ func (r Routing) deleteClusterBackupSchedule() http.Handler {
 	)
 }
 
+func (r Routing) listProjectCBSL() http.Handler {
+	return httptransport.NewServer(
+		endpoint.Chain(
+			middleware.TokenVerifier(r.tokenVerifiers, r.userProvider),
+			middleware.UserSaver(r.userProvider),
+		)(storagelocation.ListCBSLEndpoint(r.userInfoGetter, r.backupStorageProvider, r.projectProvider)), storagelocation.DecodeListProjectCBSLReq, handler.EncodeJSON, r.defaultServerOptions()...,
```

Comment:
So from my understanding of the api, this should not work: 
1. You will need to add the corresponding Decode Function from your handler.go as the httpDecodeRequestFunc  argument to httptransport.NewServer
2. You will need the handler.EncodeJSON as the httpEncodeRequestFunc argument to httptransport.NewServer

For example have a look here https://github.com/kubermatic/dashboard/blob/27df2fc9d1c9873dc07a019eff29e1a4481e21dc/modules/api/pkg/handler/v2/routes_v2.go#L7837-L7838

Reply:
Actually, the endpoint is working, and I can list the CBSL from the UI. However, I'm not quite sure what issue you're pointing to because the `storagelocation.DecodeListProjectCBSLReq` and `handler.EncodeJSON` are there.